src/Symfony/Component/Validator/Constraints/Range.php

```diff
@@ -34,7 +34,7 @@ class Range extends Constraint
 
     public $minMessage = 'This value should be {{ limit }} or more.';
     public $maxMessage = 'This value should be {{ limit }} or less.';
-    public $invalidMessage = 'This value should be a valid number.';
+    public $invalidMessage = 'This value should be a valid number or a valid datetime.';
```

Comment:
Changing the message here is a BC break (the fact that you needed to update the existing translation messages indicates it). We need to update the translation in the `validators.en.xlf` file instead.

Reply:
@xabbuh As I understand, all translations can be modified, as long as only the `target` field is modified and not the `source` field?